### ISSUE TITLE
djangorestframework-jwt is not maintained anymore

### ISSUE THREAD

**Issue by submarcos** [Terralego/django-terra-accounts#14]
Need to switch on drf-jwt, an official fork

https://github.com/Styria-Digital/django-rest-framework-jwt

**Comment by nahuelange** [comment]
Yes, or https://github.com/davesque/django-rest-framework-simplejwt that seems to be a more complete implementation of JWT.
